core/templates/services/exploration-html-formatter.service.ts

```diff
@@ -35,9 +52,50 @@ import { IInteractionCustomizationArgs } from
 })
 export class ExplorationHtmlFormatterService {
   constructor(
-    private camelCaseToHyphens: CamelCaseToHyphensPipe,
-    private extensionTagAssembler: ExtensionTagAssemblerService,
-    private htmlEscaper: HtmlEscaperService) {}
+      private camelCaseToHyphens: CamelCaseToHyphensPipe,
+      private extensionTagAssembler: ExtensionTagAssemblerService,
+      private htmlEscaper: HtmlEscaperService
+  ) {}
+  /**
+   * Convert SubtitledHtml to html string and SubtitledUnicode to unicode
+   * string.
+   * @param caValues customization argument values
```

Comment:
Write in complete sentences please. E.g.: 

`@param caValues The customization arg values whose fields should be converted.`